src/test/java/tech/jhipster/lite/generator/buildtool/gradle/domain/gradle/GradleDomainServiceTest.java

```diff
@@ -55,4 +58,15 @@ void shouldAddGradleWrapper() {
     verify(projectRepository, times(4)).add(any(Project.class), anyString(), anyString(), anyString());
     verify(projectRepository, times(2)).setExecutable(any(Project.class), anyString(), anyString());
   }
+
+  @Test
+  void shouldAddDependency() {
+    Project project = tmpProject();
+    Dependency dependency = Dependency.builder().groupId("org.springframework.boot").artifactId("spring-boot-starter").build();
+
+    gradleDomainService.addDependency(project, dependency);
+
+    String newTextExpected = ("implementation(\"org.springframework.boot:spring-boot-starter\")\n" + GRADLE_NEEDLE_DEPENDENCY).indent(2);
+    verify(projectRepository).replaceText(project, "", "build.gradle.kts",REGEXP_SPACE_STAR + GRADLE_NEEDLE_DEPENDENCY, newTextExpected);
```

Comment:
same, prettier should detect this format:
```suggestion
    verify(projectRepository).replaceText(project, "", "build.gradle.kts", REGEXP_SPACE_STAR + GRADLE_NEEDLE_DEPENDENCY, newTextExpected);
```

Reply:
Same thing, prettier didn't corrected it when I ran `npm run prettier:format`

Reply:
Yes there is a bug with prettier, I need to open a ticket to track it